src/frontend/web_application/src/styles/object/o-avatar.scss

```diff
@@ -2,7 +2,8 @@ $m-avatar__size: (
   small:  2rem,
   medium: 3rem,
   large:  4rem,
-  xlarge: 6rem
+  xlarge: 6rem,
+  xxlarge: 8rem
```

Comment:
From a technical point of view I agree, 3 sizes should be enough.
from a graphical point of view, 1rem can make the difference ^^